src/components/search/GlobalSearchField.js

```diff
@@ -579,8 +579,7 @@ function GlobalSearchField(props) {
         ]);
 
         setTeamSearchKey([SEARCH_TEAMS_QUERY, { searchTerm }]);
-        const isLoggedIn =
-            userProfile?.id !== null && userProfile?.id !== undefined;
+        const isLoggedIn = userProfile?.id;
```

Comment:
I think this needs to be a boolean?
```suggestion
        const isLoggedIn = !!userProfile?.id;
```